packages/jsii-dotnet-runtime-test/test/Amazon.JSII.Runtime.IntegrationTests/ComplianceTests.cs

```diff
@@ -1291,12 +1291,36 @@ public void StructsAreUndecoratedOntheWayToKernel()
         {
             var json = JsonFormatter.Stringify(new StructB {RequiredString = "Bazinga!", OptionalBoolean = false});
             var actual = JObject.Parse(json);
-            
+
             var expected = new JObject();
             expected.Add("RequiredString", "Bazinga!");
             expected.Add("OptionalBoolean", false);
 
             Assert.Equal(expected, actual);
         }
+
+        [Fact(DisplayName = Prefix + nameof(PureInterfacesCanBeUsedTransparently))]
+        public void PureInterfacesCanBeUsedTransparently()
+        {
+            var expected = new StructB { RequiredString = "It's Britney b**ch!" };
+            var del = new StructReturningDelegate(expected);
+            var consumer = new ConsumePureInterface(del);
+            Assert.Equal(expected.RequiredString, consumer.WorkItBaby().RequiredString);
+        }
+
+        private sealed class StructReturningDelegate: DeputyBase, IStructReturningDelegate
```

Comment:
Why do you need to extend `DeputyBase`??

Reply:
Due to #1029.

Reply:
Will you remember to remove?